projects/cashmere/src/lib/app-switcher/app-switcher.component.scss

```diff
@@ -39,9 +39,6 @@
                 & > .thumbnail-img {
                     width: 100px;
                     height: 100px;
-                    background-size: contain;
```

Comment:
Can you confirm that this is not a regression of #503 ?

Reply:
see above